src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private int firstLeg;
+    private int secondLeg;
```

Comment:
Consider renaming 'secondLeg' to 'height' for better readability and understanding 📐